server/index.js

```diff
@@ -34,6 +35,12 @@ function main() {
     this.set('Timing-Allow-Origin', '*');
     yield next;
   });
+
+
+  app.use(cors({
```

Comment:
Without this, the liveblog page refused to load liveblog.js.

What's the correct way to make it route-specific?